pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go

```diff
@@ -593,15 +593,10 @@ func (r *Reconciler) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 			Fn:           flow.TaskFn(botanist.CleanupOrphanedDNSRecordSecrets).DoIf(!o.Shoot.HibernationEnabled),
 			Dependencies: flow.NewTaskIDs(deployInternalDomainDNSRecord, deployExternalDomainDNSRecord, deployOwnerDomainDNSRecord, deployIngressDomainDNSRecord),
 		})
-		vpnLBReady = g.Add(flow.Task{
-			Name:         "Waiting until vpn-shoot LoadBalancer is ready",
-			Fn:           flow.TaskFn(botanist.WaitUntilVpnShootServiceIsReady).SkipIf(o.Shoot.HibernationEnabled || o.Shoot.ReversedVPNEnabled),
```

Comment:
Can you delete this function now? https://github.com/gardener/gardener/blob/fc1fdfcfb9f029a88005ec3f98178c68d79f49f7/pkg/operation/botanist/waiter.go#L47-L52

Reply:
Done.